gym/envs/toy_text/taxi.py

```diff
@@ -1,11 +1,13 @@
 import sys
 from contextlib import closing
 from io import StringIO
+from os import path
 from typing import Optional
 
 import numpy as np
 from gym import Env, spaces, utils
 from gym.envs.toy_text.utils import categorical_sample
+import pygame
```

Comment:
I think we could defer the pygame import to the `_render_gui()` method, so that #2691 is preemptively solved (similar to #2712)